src/Simulation/EntryPointDriver/Microsoft.Quantum.EntryPointDriver.csproj

```diff
@@ -4,9 +4,8 @@
   
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Microsoft.Quantum.CsharpGeneration.EntryPointDriver</RootNamespace>
-    <AssemblyName>Microsoft.Quantum.CsharpGeneration.EntryPointDriver</AssemblyName>
```

Comment:
It seems to be based on the name of the project file.

Reply:
You're right, it actually is based on `AssemblyName`. By default that is the same name as the project file, but changing the assembly name will also change the package reference.

Reply:
That's pretty cool! I'd suggest to keep the explicit AssemblyName property then (same as for all other projects).

Reply:
> (same as for all other projects)

Do you mean that I should add an `AssemblyName` so that EntryPointDriver is consistent with the other projects? I searched the repository, and out of 20 projects, I found only 4 that use an `AssemblyName` property, and only 2 of them where the `AssemblyName` is the same as the `.csproj`/`.fsproj` file name.

Reply:
Looks like there is also a `PackageId` property, which can override the `AssemblyName` property.